### ISSUE TITLE
[1.10.2] Roots boss drops are blocked by Sponge

### ISSUE THREAD

**Issue by ProsperCraft** [SpongePowered/SpongeForge#1746]
Remember this issue? https://github.com/SpongePowered/SpongeForge/issues/857

We are having this issue with the roots boss- The Guardian of Sprites will not drop the otherworld essence when sponge is running.

When sponge is enabled we killed it 12 times with otherworld essence dropping only once.

When sponge was disabled we got between 8-11 otherworld essence every single kill.  Also there are charms that drop that are more common to drop with sponge disabled.

**I am currently running...**
I am currently running...
Minecraft: 1.10.2
SpongeAPI: 5.2.0-SNAPSHOT-c675e80
SpongeForge: 1.10.2-2281-5.2.0-BETA-2597
Minecraft Forge: 12.18.3.2316
Java version: openjdk version "1.8.0_141"
OpenJDK Runtime Environment (build 1.8.0_141-b16)
OpenJDK 64-Bit Server VM (build 25.141-b16, mixed mode)

Operating System: Centos 7

[ X] Linux (Ubuntu, Debian, Fedora, etc)

Plugins/Mods:
[18:23:20] [Server thread/INFO]: Plugins (130): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, Aroma1997Core, SpongeAPI, SpongeForge, CoFH ASM, FoamFixCore, ccl-entityhook, Nucleus Mixins, IndustrialCraft 2, Advanced Solar Panels, Just Enough Items, AppleCore, AppleSkin, ArchitectureCraft, Aroma1997Core|Helper, Aroma1997's Dimensional World, Astral Sorcery, Bad Wither No Cookie!, Baubles, BD Lib, Bed Bugs, BetterBeginnings, Better Builder's Wands, [BlockNuke], Guide-API, Blood Magic: Alchemical Wizardry, Botania, CodeChicken Lib, CoFH Core, Brandon's Core, BTFU, Chest Transporter, Chisel, MCMultiPart, Chisels & Bits, Clay Bucket, CodeChicken Core, Cooking for Blockheads, CosmeticArmorReworked, CraftTweaker JEI Support, MineTweaker 3, Dimension Rules, Draconic Evolution, Elevator Mod, EnderCore, Waila, Ender IO, EnderStorage, Valkyrie Lib, Environmental Tech, Extra Utilities 2, Zero CORE, Extreme Reactors, Uranium Big Reactor, FoamFix, Forestry, FTBLib, FTBUtilities, GenDustry, Grapple Hooks, Hardcore Wither, HelpFixer, iChunUtil, Mantle, Tinkers' Construct, Immersive Engineering, Immersive Petroleum, MTLib, InitialInventory, Inventory Tweaks, Iron Chest, Just Enough Forestry Bees, JEI Bees, JourneyMap, Knob Control, libVulpes, LunatriusCore, MineTweakerRecipeMaker, Mod Tweaker, numina, MachineMuse's Modular Powersuits, RebornCore, TechReborn, More Bees, Morpheus, Mystical Agriculture, No Fist, Nomadic Tents, Pam's HarvestCraft, PortalGun, Progressive Automation, reborncore-MCMultiPart, Refined Storage, RebornStorage, Roots, Realistic Terrain Generation, SG Craft, ShetiPhian-Core, The Spice of Life, Stackie, Thermal Foundation, Thermal Expansion, Thermal Dynamics, Tomb Many Graves, WAILA Features, WAILA Plugins, WanionLib, I Can See Everything, What Are We Looking At, Woot, YABBA, EconomyLite, NuVotifier, Enjin Minecraft Plugin, LuckPerms, Luckrankup, Nucleus, MagiBridge, MMCReboot, PayDay, Prism, PwnFilter Sponge Plugin, React, SendToSpawn, VotifierListener, denseores, UniDict

[fml-server-1.zip](https://github.com/SpongePowered/SpongeForge/files/1249591/fml-server-1.zip)

There is a sprite guardian spawn egg, use that for testing.

**Comment by ghost** [comment]
Try this setting:

/sponge -g config optimizations.drops-pre-merge false

**Comment by ProsperCraft** [comment]
Already in my config- 
        drops-pre-merge=false

**Comment by ProsperCraft** [comment]
This is still happening with

[01:17:43] [Server thread/INFO]: SpongeForge
    Minecraft: 1.10.2
    SpongeAPI: 5.2.0-SNAPSHOT-c675e80
    SpongeForge: 1.10.2-2477-5.2.0-BETA-2698
    Minecraft Forge: 12.18.3.2316


Verified by loading with forge only, and the drops happen just fine.

**Comment by phit** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.
